_posts/2021-05-18-Welcome-GSoC-students.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Fortran-lang welcomes new students to Google Summer of Code 2021"
+category: newsletter
+author: Milan Curcic, Ondřej Čertík, Sebastian Ehlert, Laurence Kedward, Arjen Markus, Brad Richardson
+---
+
+We're happy to announce six students that will work on Fortran projects under
+the Google Summer of Code 2021 program:
+
+* [Aman Godara](https://github.com/aman-godara) will work on strings in the
+  [Fortran Standard Library](https://github.com/fortran-lang/stdlib). Aman's
+  mentors will be [Sebastian Ehlert](https://github.com/awvwgk) and
+  [Milan Curcic](https://github.com/milancurcic).
+* [Rohit Goswami](https://github.com/haozeke) will work on the
+  [LFortran](https://lfortran.org) compiler, specifically toward the capability
+  to compile a complex physics package. Rohit's mentor will be
+  [Ondřej Čertík](https://github.com/certik).
+* [Jakub Jelinek](https://github.com/kubajj) will work on handling compiler
+  arguments in the
+  [Fortran Package Manager](https://github.com/fortran-lang/fpm). Jakub's
+  mentors will be [Laurence Kedward](https://github.com/lkedward) and
+  [Brad Richardson](https://github.com/everythingfunctional).
+* [Chetan Karwa](https://github.com/chetankarwa) will work on the linked list
+  support in the
+  [Fortran Standard Library](https://github.com/fortran-lang/stdlib). Chetan's
+  mentors will be [Arjen Markus](https://github.com/arjenmarkus) and
+  [Milan Curcic](https://github.com/milancurcic).
+* [Thirumalai Shaktivel](https://gitlab.com/Thirumalai-Shaktivel) who will work
```

Comment:
The above 4 names aren't followed by "who". Either add it above or remove it here and in the bullet below. Other than that it all looks good.